mlir-tensorrt/tensorrt/lib/Target/TensorRTEncodingOpInterface/NetworkEncoder.cpp

```diff
@@ -222,20 +222,39 @@ nvinfer1::Permutation tensorrt::getNvInferPermutation(ArrayRef<int64_t> array) {
 static std::string getUniqueName(NvInferNetworkEncoder::NamesSet &names,
                                  std::string name) {
   static unsigned i = 0;
-  std::string uniqueName = name;
-  while (names.contains(uniqueName))
-    uniqueName = name + "_" + std::to_string(i++);
-  names.insert(uniqueName);
-  return uniqueName;
+  std::string newName = name;
+  while (names.contains(newName))
+    newName = name + "_" + std::to_string(i++);
+  names.insert(newName);
+  return newName;
```

Comment:
How does this fix the naming? It seems like it's just renaming a variable?

Reply:
you are right, this must have been some mege issue in our internal branch where the code was
```
while (names.contains(name))
    name = name + "_" + std::to_string(i++);
```
before making this change, which was causing the behavior I described.  I can undo this change for this function.